packages/lmi/tests/test_cost_tracking.py

```diff
@@ -212,10 +212,36 @@ async def test_multiple_callbacks_with_one_failing(self, caplog):
             cost_tracking_ctx(),
             patch("litellm.cost_calculator.completion_cost", return_value=0.01),
         ):
-            GLOBAL_COST_TRACKER.record(mock_response)
+            await GLOBAL_COST_TRACKER.record(mock_response)
 
             failing_callback.assert_called_once_with(mock_response)
             succeeding_callback.assert_called_once_with(mock_response)
 
             assert "Callback failed during cost tracking" in caplog.text
             assert GLOBAL_COST_TRACKER.lifetime_cost_usd > 0
+
+    @pytest.mark.asyncio
+    async def test_async_context_with_stream_wrapper(self):
+        mock_stream = MagicMock()
+        mock_response = MagicMock(cost=0.01)
+        mock_stream.__anext__ = AsyncMock(return_value=mock_response)
+
+        wrapper = TrackedStreamWrapper(mock_stream)
+
+        callback_calls: list[Any] = []
```

Comment:
```suggestion
        callback_calls = []
```

Doing `list[Any]` defeats type inference, effectively dumbing `mypy`. Let's let our tests be typed, so `mypy` can check here too